.rubocop.yml

```diff
@@ -16,6 +16,9 @@ Lint/Debugger:
   Exclude:
   - lib/cc/cli/console.rb
 
+Lint/AssignmentInCondition:
```

Comment:
We're ok with it. IIRC the rule specifies that they have to be surrounded by parentheses, so the line below would be 

```ruby
if (fingerprint = json["fingerprint"])
```

That should not set off the cop.